### PR TITLE
create remake.yml file

### PR DESCRIPTION
**Pull request by ellenbechtel**
Made pull request as per issue #6

### CODE REVIEW

File: remake.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+sources:
+  - code.R
+
+targets:
+  all:
+    depends: figure_1.png
```
[the remark this answers is not in the document]
Comment (ellenbechtel) [reply remake.yml]: I think I changed it correctly?  Let's see.

File: remake.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+  model_RMSEs.csv:
+    command: download_data(target_name)
```
[the remark this answers is not in the document]
Comment (ellenbechtel) [reply remake.yml]: I think so... let's try this edit.

File: remake.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+  processed:
+    command: process_data("model_RMSEs.csv")
```
[the remark this answers is not in the document]
Comment (ellenbechtel) [reply remake.yml]: I think I did that too with this new edit.

File: remake.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+  figure_1.png:
```
[the remark this answers is not in the document]
Comment (ellenbechtel) [reply remake.yml]: Done

**Review by jordansread — APPROVED** [review]
Looks good.

**Pull request merged**